### PR TITLE
Update typography section to match new brand guidelines

### PR DESCRIPTION
**Pull request by MilesRJohnson**
The aim of this PR is to update the typography section so it matches the new brand guidelines. 

__Before__

type-before

__After__

type-after

### CODE REVIEW

File: source/assets/stylesheets/base/_variables.scss [hunk 1]
```diff
@@ -11,9 +11,12 @@ $large-screen: 900px;
 $sans-serif: "Calibre", $font-stack-system;
 $base-font-family: $sans-serif;
 $heading-font-family: $sans-serif;
+$serif: "Tiempos", $font-stack-system;
```
Comment (tysongach) [inline source/assets/stylesheets/base/_variables.scss]: Should we use a serif as a fallback font, instead of the sans-serif system fonts?

**Comment by tysongach** [comment]
Following up on my comment above…I believe we have an S3 bucket for asset usage, like that. I believe @mjankowski was talking about this yesterday on another thread. Alternatively, we could create a private repo to house the fonts and publish them via GitHub Pages, like https://github.com/thoughtbot/bourbon.io-fonts

**Comment by MilesRJohnson** [comment]
Okey dokey, this makes sense. Is adding the items to an S3 bucket as simple as logging into a thoughtbot aws account and uploading the files there to reference? Sorry to ask, but if you or @mjankowski could give me some guidance on that process, I can try to remedy both this and the issue from [#51](https://github.com/thoughtbot/presskit/pull/51) regarding the guidelines.

**Comment by tysongach** [comment]
Here‘s info on the S3 bucket: https://github.com/thoughtbot/handbook/blob/master/marketing/images.md

I assume it’s fine to use for fonts, as well, and that fonts don’t require some more config or setup…but I’ve not tested.

**Comment by mjankowski** [comment]
Yes, I think using the existing "images" bucket in S3 - and adding a well-named sub-folder makes sense.

I just created a "presskit" folder - https://s3.console.aws.amazon.com/s3/buckets/thoughtbot-images/presskit/?region=us-east-1&tab=overview - feel free to use that to upload static assets related to this project.

I believe that files added there will show up at `https://images.thoughtbot.com/presskit/***` - but we might need to wrangle permissions after you upload a few things to confirm.

**Comment by MilesRJohnson** [comment]
Thanks @mjankowski! I had success with the pdf, just need to sort out the font files.

**Comment by MilesRJohnson** [comment]
@mjankowski sorry for the delay on this PR, I was having issues retrieving the font files and running into CORS issues. With some help from @nickcharlton we were able to temporarily modify the CORS permissions so that I could confirm that the fonts were being retrieved and applied properly. I have returned the permissions to normal so once merged we will need to confirm that the files are being retrieved ok.

**Comment by tysongach** [comment]
Based on the [Netlify deploy preivew](https://deploy-preview-53--thoughtbot-presskit.netlify.com/), it appears that the fonts are not working…is this the CORS issue you’re talking about?

**Comment by mjankowski** [comment]
I attempted to change the CORS configuration on S3 to allow these requests, but couldn't sort it out in the ~5 mins I allowed. If anyone has more experience with that and can point at useful howto, please do.

**Comment by nickcharlton** [comment]
I'd been meaning to take a look at this since trying to help out @MilesRJohnson.

It wasn't a CORS problem, that's configuration is correct. The problem was the object permissions for the newly added items weren't marking those as publicly accessible in the same way as other objects in this S3 bucket are.

For every item in `thoughtbot-images/presskit/fonts`, I've marked them as public and now the Netlify deploy preview is working correctly.

**Comment by tysongach** [comment]
Nice - thanks @nickcharlton!

**Pull request merged**